README.md

```diff
@@ -177,6 +185,12 @@ For the replacement of `racadm racreset`, the optional argument `--racreset` was
 ./src/badfish.py -H mgmt-your-server.example.com -u root -p yourpass --racreset
 ```
 
+### BIOS factory reset
+You can restore BIOS default settings by calling Badfish with the option `--bios-reset`.
```

Comment:
I'd put something here like:

**WARNING** Use this carefully, vendor defaults differ and may be disruptive.  Do not use this in the Scale Lab or ALIAS.